spec/unit/type/component_spec.rb

```diff
@@ -43,7 +43,7 @@
     end
 
     it "should produce the class name even for the class named main" do
-      expect(component.new(:name => "Class[main]").pathbuilder).to eq(["Main"])
+      expect(component.new(:name => "Class[main]").pathbuilder).to eq(['main'])
```

Comment:
According to https://docs.puppet.com/puppet/4.10/lang_reserved.html#reserved-class-names, `main` is a reserved class name, though attempts to use it produce a duplicate resource declaration error, which isn't what I was expecting. I assume it's because the implicit `:main` class resolves to `Class[main]{:name=>"main"}`:

```puppet
cat main.pp
class main {
  notify { 'hi': }
}
include main
```

```
$ bx puppet apply main.pp
...
Error: Duplicate declaration: Class[Main] is already declared; cannot redeclare
```